tests/Utilities/MakeWithRandomValues.hpp

```diff
@@ -180,7 +180,8 @@ template <typename T, typename UniformRandomBitGenerator,
 T make_with_random_values(
     const gsl::not_null<UniformRandomBitGenerator*> generator,
     const gsl::not_null<RandomNumberDistribution*> distribution) noexcept {
-  T result{std::numeric_limits<tt::get_fundamental_type_t<T>>::signaling_NaN()};
+  T result
```

Comment:
make this change only in one commit. And in whichever commit the real change is, make sure that it calls `fill` for container types, and just constructs with `NaN` for fundamental types.